### PR TITLE
script: Bump to Fedora 34 & update Gems for Ruby 3.x

### PR DESCRIPTION
**Pull request by garrett**

### CODE REVIEW

**Comment by garrett** [comment]
There's nothing to preview, so I made a bogus testing commit on my main branch. If there's a CONGRATULATIONS! under "Introducing Cockpit" then GitHub pages works with the new Gems and this should be fine to merge.

https://garrett.github.io/cockpit-project.github.io/

...and there is!

image

File: _scripts/toolbox-create [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/bash -e
 
 echo "## Creating website container..."
-toolbox create -c cockpit-website -r 33
+toolbox create -c cockpit-website -r 34
```
Comment (garrett) [inline _scripts/toolbox-create:4]: Making the decision to lock the container to specific Fedora releases paid off. Ruby was upgraded to 3.x in Fedora 34 and it broke things, so keeping it on Fedora 33 for a while until investigating what happened was the right choice.

**Comment by garrett** [comment]
Since it has been 19 days, I could see about bumping up the versions again, just to merge in something fresher.

**Review by martinpitt — APPROVED** [review]
Thanks!

**Pull request merged**